### PR TITLE
Introduce force in uploader and cp command

### PR DESCRIPTION
**Pull request by paweljw**

### CODE REVIEW

File: spec/integration/ccs/uploader_spec.rb [hunk 1]
```diff
@@ -16,16 +16,23 @@
...
                  headers: { 'Content-Type' => 'application/json' })
     stub_request(:post, 'http://example.com/a/b/d.yml')
+      .with(body: a_string_including('"force":"false"'))
       .to_return(status: 409,
                  body: { message: 'Conflict', status: '409' }.to_json,
                  headers: { 'Content-Type' => 'application/json' })
+    stub_request(:post, 'http://example.com/a/b/d.yml')
+      .with(body: a_string_including('"force":"true"'))
+      .to_return(status: 200,
+                 body: { message: 'Created', status: '200' }.to_json,
```
Comment (tkowalewski) [inline spec/integration/ccs/uploader_spec.rb:26]: Maybe message should be 'Ok'?

File: spec/integration/ccs/document_spec.rb [hunk 2]
```diff
@@ -19,12 +19,21 @@
       .to_return(status: 201,
                  body: { message: 'Created', status: '201' }.to_json,
                  headers: { 'Content-Type' => 'application/json' })
+    stub_request(:post, 'https://api.occson.com/a/b/c.txt')
+      .with(body: a_string_including('"force":"true"'))
+      .to_return(status: 200,
+                 body: { message: 'Created', status: '200' }.to_json,
```
Comment (tkowalewski) [inline spec/integration/ccs/document_spec.rb:25]: Maybe message should be 'Ok'?

File: exe/ccs [hunk 3]
```diff
@@ -42,6 +42,10 @@ when 'cp'
       options['passphrase'] = v
     end
 
+    option_parser.on('-f', '--[no-]force', 'Overwrite remote documents when uploading') do |v|
+      options['passphrase'] = v
```
Comment (tkowalewski) [inline exe/ccs:46]: Maybe there is wrong key? Should be force?

File: exe/ccs [hunk 4]
```diff
@@ -57,6 +61,8 @@ when 'cp'
     option_parser.separator '    ccs cp ccs://workspace-name/0.1.0/path/to/file.yml /local/path/to/file.yml'
     option_parser.separator '  Upload local file'
     option_parser.separator '    ccs cp /local/path/to/file.yml ccs://workspace-name/0.1.0/path/to/file.yml'
+    option_parser.separator '  Upload local file, overwriting remote document if any'
```
Comment (tkowalewski) [inline exe/ccs:64]: Can we add this information also to README?

File: spec/integration/ccs/document_spec.rb (hunk 2 above)
Comment (paweljw) [reply spec/integration/ccs/document_spec.rb:25]: Ah, the api doesn't even return these on success - I'll mock more realistic responses there. Uploader only reads response status anyway.

File: exe/ccs (hunk 3 above)
Comment (paweljw) [reply exe/ccs:46]: Oops! My bad

**Review by tkowalewski — APPROVED** [review]
👍 Great job!

**Pull request merged**